### PR TITLE
Set up apollo

### PR DESCRIPTION
**Pull request by nzambonivergara**
### Features
- Set up Apollo Client
- Create utils folder with `graph_queries.js` and `Models.tsx`
- Create assets folder with background pictures
- Deleted default JSX and styles that comes with create react app

### Refactors 
- NA

### Issues and Technical Debt
- NA

### Other Comments, Concerns, or Questions
- NA

### Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature work
- [ ] New and existing unit tests pass locally with my changes
- [ ] I ran Lighthouse and WAVE accessibility audits and fixed errors/warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by TannerDale — COMMENTED** [review]
Gonna have to review the whole app a lot to fully grasp it but it looks good! I saw the CI failed, is it fully set up?

**Comment by nzambonivergara** [comment]
Thank you, Tanner! 
It was initially failing because the user variable in App wasn't being used, and all warnings will be treated as errors in the build. I used the variable in the render method and that fixed that issue.
Then, it was failing to deploy. The reason it was failing to deploy is that we had merged our cicd workflow branch into main, but we had not redeployed so the latest version of main was not deployed. I ran npm run deploy from main and then re-ran the workflow in the set-up-apollo branch and it is passing now.

File: src/App.tsx [hunk 1]
```diff
@@ -1,23 +1,36 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+      <header className="header">
+        <h1>GlowUp</h1>
+        { user && <h2>Welcome { user.username }</h2>}
```
Comment (nzambonivergara) [inline src/App.tsx:29]: This is here temporarily to prevent the warning of the user variable not being used.

File: src/utils/Models.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+
+interface Mood {
+    id: string
+    mood: number
+    description: string
+    createdAt: string
+}
+
+// export interface Habit {
```
Comment (nzambonivergara) [inline src/utils/Models.tsx:9]: Usually I would not leave commented out code, but I believe we are going to need this very soon.

File: src/App.tsx (hunk 1 above)
Comment (woaishenzhen) [inline src/App.tsx:29]: I believe this is part of the fix :)

**Review by woaishenzhen — APPROVED** [review]

**Pull request merged**